### PR TITLE
Edited files to migration browserid to bidpom

### PR DESCRIPTION
**Pull request by carcablop**

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -30,13 +30,19 @@ def stored_users(variables):
...
+#     login_pg.go_to_login_page()
+#     login_pg.login(existing_user['email'], existing_user['password'])
 
 @pytest.fixture(scope='function')
-def login(request, base_url, selenium, existing_user):
-    from pages.login_page import MozTrapLoginPage
-    login_pg = MozTrapLoginPage(base_url, selenium)
-    login_pg.go_to_login_page()
-    login_pg.login(existing_user['email'], existing_user['password'])
+def login(request,base_url,selenium, existing_user):
```
Comment (davehunt) [inline conftest.py]: arguments should be separated by `, ` to pass flake8 linting

File: conftest.py [hunk 2]
```diff
@@ -30,13 +30,19 @@ def stored_users(variables):
...
+    selenium.find_element_by_id('login').click()
```
Comment (davehunt) [inline conftest.py]: Please modify the `MozTrapLoginPage` to avoid using Selenium directly.

**Comment by davehunt** [comment]
Please run your changes through flake8 to fix the Travis CI issue, and comment here when you've pushed an update.

**Comment by rbillings** [comment]
@carcablop Please run your changes through flake8 to fix the Travis CI issue, and comment here when you've pushed an update.

**Comment by davehunt** [comment]
Due to lack of activity, let's close this and open it up for @stephendonner to work on.

**Pull request closed without merging**